src/components/Link/Link.Props.ts

```diff
@@ -12,6 +12,11 @@ export interface ILinkProps extends React.HTMLProps<HTMLLabelElement> {
    *  The props of pop up window.
    */
   popupWindowProps?: IPopupWindowProps;
+
+  /**
+   * Function to call when the card is clicked.
+   */
+  onClick?: (ev?: React.MouseEvent) => void;
```

Comment:
I don't think this is necessary. This component should be receiving HTMLAnchorElement props. I'm OK with changing the inherited type to that, or HTMLButtonElement + href + target.

But I don't even think that's necessary because I'm pretty sure the HTMLLabelElement also allows onClick already. You don't need to add it.